src/main/java/seedu/address/model/person/github/repo/Repo.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person.github.repo;
+
+/**
+ * Class for each individual repository for a specified GitHub.
+ */
+public class Repo {
+    private String repoName;
+    private String language = "";
+    private int numberForks = 0;
+    private String license = "";
+    private String lastUpdated = "";
+
+    public Repo(String repoName) {
+        this.repoName = repoName;
+    }
```

Comment:
If we changed it to immutable, the method descriptor would be
```suggestion
    public Repo(String repoName, String language, int numOfForks, String license...) {
        //...
    }
```

Reply:
Have incorporated the suggested change in latest commit.